src/main/scala/mesosphere/chaos/rest/RestModule.scala

```diff
@@ -0,0 +1,34 @@
+package mesosphere.chaos.rest
+
+import javax.servlet.http.HttpServlet
+
+import com.codahale.metrics.MetricRegistry
+import com.codahale.metrics.servlets.MetricsServlet
+import mesosphere.chaos.http.HttpService
+
+import mesosphere.chaos.rest.impl.{LogConfigServlet, ServiceStatusServlet, PingServlet}
+
+/**
+  * Created by alex on 21/01/16.
+  */
+class RestModule(httpService: HttpService, metricRegistry: MetricRegistry) {
+  // Override these in a subclass to mount resources at a different path
+  val pingUrl = "/ping"
+  val metricsUrl = "/metrics"
+  val loggingUrl = "/logging"
+  val statusUrl = "/status"
+  val statusCatchAllUrl = "/status/*"
+
+  // Initializing servlets
+  lazy val pingServlet: HttpServlet = new PingServlet
+  lazy val metricsServlet: HttpServlet = new MetricsServlet(metricRegistry)
+  lazy val logConfigServlet: HttpServlet = new LogConfigServlet
+  lazy val serviceStatusServlet = new ServiceStatusServlet(new ServiceStatus())
```

Comment:
Cool that you replaced this functionality - I still think we should remove the service status.